Data/System/strings.csv

```diff
@@ -433,15 +433,16 @@ Mouse controls,Commandes souris,Maus konfigurieren,Configurar el ratón,Configur
 No gamepad detected!,Aucune manette détectée!,Kein Controller erkannt!,,Nessun gamepad rilevato!,Ingen handkontroll hittades!,Geen gamecontroller gedetecteerd!
 Force feedback,Force des vibrations,Vibrationsstärke,Fuerza de vibración,Forza delle vibrazioni,Vibrationsstyrka,
 Off,Aucune
-Low,Faible
-Medium,Moyenne
-High,Élevée
+Low,Faible,,,Basso
+Medium,Moyenne,,,Medio
+High,Élevée,,,Alto
```

Comment:
These go with "forza delle vibrazioni" - in French it sounded more natural to use the feminine form here because "force" is feminine, I suppose like "forza".  What do you reckon works best for Italian?

Reply:
Done. I wasn't sure whether is was used in other occasions.